wincrypto/src/lib.rs

```diff
@@ -0,0 +1,39 @@
+#[macro_use]
```

Comment:
use tracing::{info, warn, debug};

instead

Reply:
I'm matching str0m here.

Reply:
str0m is out of date @algesten  :)

Reply:
It's just personal preference. The log/debug macros are used a bit of everywhere and I think it's irritating to import them everywhere. Hence I keep doing `#[macro_use]`

Reply:
Resolving, as this should match the overall str0m style.